internal/pkg/manifest/validate.go

```diff
@@ -843,6 +846,16 @@ func (ip IPNet) validate() error {
 
 // validate returns nil if NetworkLoadBalancerConfiguration is configured correctly.
 func (c NetworkLoadBalancerConfiguration) validate() error {
+	nlbRoutingRules := c.NLBRoutingRules()
+	for _, rule := range nlbRoutingRules {
+		if err := rule.validate(); err != nil {
+			return fmt.Errorf("validate nlb routing rule; %s", err.Error())
```

Comment:
😮 do we do this anywhere? this seems so strange
```suggestion
			return fmt.Errorf("validate nlb.listener[%i]: %v", i, err)
```

Reply:
I think that just happened by mistake 🙈